src/components/features/CompareProducts/CompareProducts.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import styles from './CompareProducts.module.scss';
+import { faWindowClose } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Button from '../../common/Button/Button';
+import { getCompare, toggleProductCompare } from '../../../redux/productsRedux';
+
+const CompareProducts = () => {
+  const dispatch = useDispatch();
+  const compare = useSelector(state => getCompare(state));
+
+  const handleRemove = (e, id) => {
+    e.preventDefault();
+    dispatch(toggleProductCompare(id));
+  };
+
+  return (
+    <div className={'row align-items-end justify-content-center ' + styles.wrapper}>
```

Comment:
Przydałby się headline, aby odseparowac element.

Reply:
co masz na myślisz pisząc `headline`?

Reply:
Np tak jak na górze komponentu NewFurniture (nazwa sekcji), myślę że warto by jakoś odseparować ten komponent.

Reply:
Wrzuć kod pokazujący o co chodzi bo nadal nie ogarniam

Reply:
Tak chyba będzie łatwiej.

![image](https://user-images.githubusercontent.com/93682845/174454027-c5a1a03f-002d-42d5-b0fd-c4e04fdddff7.png)

Reply:
On się nie zatrzymuje tylko jest zawsze na srodku. Jak dodamy nagłówek to będą sytuace że masz nagłówek "New furniture" a pod nim zaraz `Compare`

Reply:
Więc moim zdaniem bez headline, zamiast tego dodałbym tło na całą szerokość strony, żeby pod produktami nie było widać innych sekcji. Możesz też pod futerem zrobić miejsce na te produkty, żeby faktycznie nie został footer zasłonięty

Reply:
Przekonałeś mnie.

Reply:
Ciemne czy jasne tło? :D